legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-logical-plan/src/main/java/org/finos/legend/engine/persistence/components/ingestmode/UnitemporalDeltaAbstract.java

```diff
@@ -39,6 +41,8 @@ public interface UnitemporalDeltaAbstract extends IngestMode, TransactionMilesto
 
     Optional<String> dataSplitField();
 
+    Optional<List<OptimizationFilter>> optimizationFilters();
```

Comment:
Can we remove the Optional? We should just handle empty collection with the right behavior (Optional<Collection> in general is overkill)

Reply:
Sure Will do that